app/serializers/team_serializer.rb

```diff
@@ -21,5 +21,5 @@
 class TeamSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :private
 
-  has_many :folders
+  has_many :folders, serializer: FolderMinimalSerializer
```

Comment:
Same thing as above. We never want any attribute not in the MinimalSerializer, when accessing a folder through team?